### DISCUSSION TITLE
What is JavaScript?

### DISCUSSION THREAD

**Discussion by xlgmokha** [CodeChica/plus-plus#32] [General]
Is there an easier way to remember what this JavaScript thing is?

**Comment by daitanfutekina** [comment]
the code that gives something a function, it can make things move, interact, and do the logistics or calculations if you are doing java in the back of a site as well

**Comment by superrrk** [comment]
Maybe an easier way to remember is by using the JavaScript acronym as JS to mean "Jumping Sites" because JavaScript gives the site the functionality and dynamic aspects of websites. Jumping icons, jumping to other web pages with buttons, and jumping to the next picture is all included in JS!

**Comment by IxchelO** [comment]
JavaScript is used to make the product work or a specific part work in the product.

**Comment by IxchelO** [comment]
Javascript is code that allows your specific wants to work and continue! c:

**Comment by Ariadnaqlla** [comment]
interactive elements for web pages enhancing the user experience

**Comment by code2417** [comment]
JavaScript is a programming language used to make web pages interactive. Search engines, social media and phone apps would not be possible without it.

**Comment by amisadaib** [comment]
JavaScript gives websites the function and dynamic as a website.

**Comment by NeftalynBarrera** [comment]
JavaScript is a programming language that gives the website function.
